### DISCUSSION TITLE
How can I clone a strapi project?

### DISCUSSION THREAD

**Discussion by peteresteris** [strapi/strapi#6285] [Legacy Topics]
Hi pals
I had an accident with my virtualbox machine and I lost my strapi project. I could recover the project directory. I tried to create another VBox and an empty strapi project using the same name. How can I use all the api structure for recovering my project?
Sorry, my Ubuntu Virtual hadn't access to internet and I cannot upload the strapi project.
Could anyone help me please?

**Comment by robertpiosik** [comment]
If you've restored the project directory, what's the problem :) 

API structure definitions live in `/api` folder.

**Comment by derrickmehaffy** [comment]
If you deleted the virtual drive associated with the machine then there is no "good" means of recovery
